examples/replication/hydra.yaml

```diff
@@ -0,0 +1,15 @@
+# Define a replica set
+# to support multiple replica sets use multiple config  files
```

Comment:
```suggestion
```

Lets rather make a seperate example (not in this mr though)

Reply:
Sure later.